watch/queue/queue.go

```diff
@@ -0,0 +1,152 @@
+package queue
+
+import (
+	"container/list"
+	"fmt"
+	"sync"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/go-events"
+)
+
+func ErrQueueFullCloseFailed(err error) error {
```

Comment:
This seems to be unused. If it's meant for external users, it seems like a strange function to expose.